studio/src/app/utils/editor/create-slides.utils.tsx

```diff
@@ -83,7 +85,7 @@ export class CreateSlidesUtils {
         });
     }
 
-    private static createSlideSplit(): Promise<JSX.IntrinsicElements> {
+    private static createSlideSplit(vertical = false): Promise<JSX.IntrinsicElements> {
```

Comment:
See my above comment, `vertical = false` should be replaced by `attributes: SlideAttributes = undefined`